README.md

```diff
@@ -1 +1,33 @@
-# to-do-list
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To Do List
+
+In this project I build a aplication called To Do List where you can create a list of your tasks, this aplication was built with plenty JavaScript to add and remove elements from the DOM and store data in the local storage.
+
+## Built With
+
+- Major languages
+- HTML, CSS, Javascript
+- Frameworks
+
+## Author
+
+👤 **Mahmoud Makhlouf**
+
+- GitHub: [@devMahmoud](https://github.com/devMahmoud)
+
+## 🤝 Contributing
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
```

Comment:
- [ ] please make sure to use [the template](https://github.com/microverseinc/readme-template) for the readme file provided by Microverse, including the links (for example to the MIT license). keep in mind that all sections of the template are mandatory except for the "Live Demo", "Run Tests" and "Deployment" sections, these are only mandatory if the specific project you're working on requires it.